### PR TITLE
Mention UI and tctl commands in Readme

### PR DESCRIPTION
**Pull request by feedmeapples**
## What was changed

Updated readme.md mentioning UI and "tctl" commands

## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 > ⚠️ This project is experimental and not suitable for production use. ⚠️
 
-Temporal CLI is a distribution of [Temporal](https://github.com/temporalio/temporal) that runs as a single process with zero runtime dependencies.
+Temporal CLI is a distribution of [Temporal](https://github.com/temporalio/temporal), [UI](https://github.com/temporalio/ui) and that runs as a single process with zero runtime dependencies. Temporal CLI additionally includes commands to operate on namespaces, workflows etc..
```
Comment (feedmeapples) [inline README.md:8]:
```suggestion
Temporal CLI is a distribution of [Temporal](https://github.com/temporalio/temporal), [UI](https://github.com/temporalio/ui) and that runs as a single process with zero runtime dependencies. Temporal CLI includes commands to operate on namespaces, workflows etc..
```

File: README.md [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 > ⚠️ This project is experimental and not suitable for production use. ⚠️
```
Comment (bergundy) [inline README.md:6]: I wouldn’t call this experimental any more, let’s change this to say that the api is still unstable.

File: README.md [hunk 3]
```diff
@@ -5,7 +5,7 @@
...
 
-Temporal CLI is a distribution of [Temporal](https://github.com/temporalio/temporal) that runs as a single process with zero runtime dependencies.
+Temporal CLI is a distribution of [Temporal](https://github.com/temporalio/temporal), [UI](https://github.com/temporalio/ui) and that runs as a single process with zero runtime dependencies. Temporal CLI includes commands to operate on namespaces, workflows etc..
```
Comment (bergundy) [inline README.md:8]: This reads weird

Comment (feedmeapples) [inline README.md:8]:
```suggestion
Temporal CLI is a single process binary that integrates Temporal server and UI suitable for development purposes. Additionally it includes commands helpful for operating on namespaces, workflows etc.. 
```

Comment (djmagee) [inline README.md:8]:
```suggestion
Temporal CLI is a single-process binary that integrates Temporal Server and UI suitable for development purposes. It also includes commands helpful for operating on Namespaces, Workflows, and other Temporal entities. 
```

Comment (flossypurse) [reply README.md:8]: "the Temporal Server" ?

File: README.md [hunk 4]
```diff
@@ -5,7 +5,7 @@
 
 > ⚠️ This project is experimental and not suitable for production use. ⚠️
 
-Temporal CLI is a distribution of [Temporal](https://github.com/temporalio/temporal) that runs as a single process with zero runtime dependencies.
+Temporal CLI is a single-process binary that integrates Temporal Server and UI suitable for development purposes. It also includes commands helpful for operating on Namespaces, Workflows, and other Temporal entities.
```
Comment (feedmeapples) [inline README.md:8]:
```suggestion
Temporal CLI is a single-process binary that integrates the Temporal Server and UI suitable for development purposes. It also includes commands helpful for operating on Namespaces, Workflows, and other Temporal entities. 
```

File: README.md (hunk 2 above)
Comment (feedmeapples) [inline README.md:6]:
```suggestion
> ⚠️ Temporal CLI's API is still a subject for changes. ⚠️
```

Comment (djmagee) [reply README.md:6]: "The APIs used by Temporal CLI are subject to change."

Or, if the warning should be a bit stronger, "Be aware that the APIs used by Temporal CLI are subject to change."

Comment (lorensr) [reply README.md:6]: @djmagee  I think the CLI command names/format is the "API" that Roey/Ruslan are talking about.

File: README.md [hunk 5]
```diff
@@ -5,7 +5,7 @@
 
 > ⚠️ This project is experimental and not suitable for production use. ⚠️
 
-Temporal CLI is a distribution of [Temporal](https://github.com/temporalio/temporal) that runs as a single process with zero runtime dependencies.
+Temporal CLI is a single-process binary that integrates the Temporal Server and UI suitable for development purposes. It also includes commands helpful for operating on Namespaces, Workflows, and other Temporal entities.
```
Comment (lorensr) [inline README.md:8]:
```suggestion
The Temporal CLI is a command-line interface for running Temporal Server and interacting with Workflows, Activities, Namespaces, and other parts of Temporal.
```

I think "single-process binary" is just relevant to people who are familiar with the old tools. I think it makes sense to craft our messaging for new users.

**Review by lorensr — APPROVED** [review]

**Pull request merged**